trashcli/_shtab.py

```diff
@@ -0,0 +1,8 @@
+FILE = None
```

Comment:
move the logic of this file in a new python module `trashcli/shell_completion.py`

Reply:
OK, I'll fix it.